doc/modules/clustering.rst

```diff
@@ -278,6 +285,64 @@ small, as shown in the example and cited reference.
    D. Sculley, *Proceedings of the 19th international conference on World
    wide web* (2010)
 
+.. _k_medoids:
+
+K-Medoids
+=========
+
+:class:`KMedoids` is related to the :class:`KMeans` algorithm. While
+:class:`KMeans` tries to minimize the within cluster sum-of-squares,
+:class:`KMedoids` tries to minimize the sum of distances between each point and
+the medoid of its cluster. The medoid is a data point (unlike the centroid)
+which has least total distance to the other members of its cluster. The use of
+a data point to represent each cluster's center allows the use of any distance
+metric for clustering.
+
+:class:`KMedoids` can be more robust to noise and outliers than :class:`KMeans`
+as it will choose one of the cluster members as the medoid while
+:class:`KMeans` will move the center of the cluster towards the outlier which
+might in turn move other points away from the cluster centre.
+
+Please note that K-Medoids is different from the K-Medians algorithm as it
```

Comment:
This note seems a bit strange here. Clarifying K-Medoids vs K-Medians is helpful but K-Medians is just K-Means with medians, right? So K-Medoids is different from K-Medians in exactly the same way as it is different from K-Means.

Reply:
Yeah, it seems like it. I'll try to reword that paragraph in a way that makes sense.

Reply:
done